VEDA/settings.py

```diff
@@ -120,6 +129,7 @@
         'rest_framework.permissions.IsAuthenticated',
     ),
     'DEFAULT_AUTHENTICATION_CLASSES': (
+        'edx_rest_framework_extensions.authentication.JwtAuthentication',
         'oauth2_provider.ext.rest_framework.OAuth2Authentication',
```

Comment:
Why are you using all of these authentication classes? You only need JWT and session. The others need to go away given that (a) we don't use them anywhere else and (b) they are a potential threat vector.

Reply:
@yro again here, should we get rid of other auth classes? is there any authentication class that should be removed here?